.changeset/good-starfishes-appear.md

```diff
@@ -0,0 +1,6 @@
+---
+'@typechain/ethers-v5': minor
+'@typechain/ethers-v5-test': minor
```

Comment:
We don't version `-test` packages as they are not published. You can simply remove this line.